pkg/hubble/exporter/api.go

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package exporter
+
+import (
+	"reflect"
+	"sort"
+
+	flowpb "github.com/cilium/cilium/api/v1/flow"
+	"github.com/cilium/cilium/pkg/hubble/observer/observeroption"
+	"github.com/cilium/cilium/pkg/time"
+)
+
+// FlowLogExporter exports hubble events to configured target file.
+type FlowLogExporter interface {
+	observeroption.OnDecodedEvent
+	// Stop stops this exporter instance from further events processing.
+	Stop() error
+}
+
+type FlowFilters []*flowpb.FlowFilter
+type FieldMask []string
+
+type flowLogConfig struct {
+	Name           string      `yaml:"name"`
```

Comment:
For this yaml tags, could we have `omitempty` as well?

Reply:
added